homeassistant/config_entries.py

```diff
@@ -145,6 +145,7 @@ class ExampleConfigFlow(config_entries.ConfigFlow):
     'mqtt',
     'nest',
     'openuv',
+    'device_tracker.owntracks_http',
```

Comment:
This is not supported.

Reply:
In the sense that it’s not working currently, or that I can’t use a ConfigFlow for the platform (assuming because of the dot)?

Reply:
We don't support config flows for platforms. It might work but that would be by accident.